### DISCUSSION TITLE
local ollama instance on different machine

### DISCUSSION THREAD

**Discussion by princinv** [logancyang/obsidian-copilot#813] [Q&A]
If this has been answered already this will come down, but i dont see anything here dealing with a local ollama instance that is not on the same machine but run on a local server.

i know i have it set up correctly with environmental variables:

```
      - OLLAMA_HOST=0.0.0.0
      - OLLAMA_ORIGINS=*
```

and i know i can send queries to it with other clients using `<localip>:11434`

API key is empty, which has worked elsewhere, but is obsidian-copilot not able to use the above base url?

**Comment by logancyang** [comment]
If you can curl it, you can add it as a custom model with your local IP as the base url.
